C-LinkedLists/src/removeEveryKthNode.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+OVERVIEW: Given a single linked list, remove every Kth node in the linked list.
+E.g.: 1->2->3->4->5 and K 2, output is 1->3->5.
+
+INPUTS: A linked list and Value of K.
+
+OUTPUT: Remove every Kth node in the linked list.
+
+ERROR CASES: Return NULL for error cases.
+
+NOTES:
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+
+struct node {
+	int num;
+	struct node *next;
+};
+
+struct node * getPrevNode(struct node *head, int K)
+{
+	struct node * temp=head;
+	while (K - 2>0 && temp!=NULL)
+	{
+		temp = temp->next;
+		K--;
+	}
+	return temp;
+}
+
+void removeItsNextNode(struct node *temp)
+{
+	if(temp->next != NULL)
+	temp->next = temp->next->next;
+	else temp->next = NULL;
+}
+
+struct node * removeEveryKthNode(struct node *head, int K) {
+	if (!head || K < 2)
+		return NULL;
+	struct node * temp = head,* t=head;
+	int t_k=0;
```

Comment:
better to have a good name than t_k. Even if it is a temporary variable, it will be good if it is expressive. Algos wise all good.

Reply:
Should i rename and commit it again?